src/lib/components/composites/input/ChipsInput.svelte

```diff
@@ -0,0 +1,214 @@
+
+            document.getElementById("chips-suggestions-" + label)?.focus();
+            selectedChipIndex = -1;
+        } else if (event.key === "ArrowLeft") {
+            event.preventDefault();
+            if (selectedChipIndex === -1) {
+                // edge case: no previous selection, so set to last element
+                selectedChipIndex = items.length;
+            }
+            selectedChipIndex = (selectedChipIndex - 1 + items.length) % items.length;
+
+            document.getElementById("chips-input-" + label)?.focus();
+            selectedSuggestionIndex = -1;
+        } else if (event.key === "ArrowRight") {
+            event.preventDefault();
+            selectedChipIndex = (selectedChipIndex + 1) % items.length;
+
+            document.getElementById("chips-input-" + label)?.focus();
+            selectedSuggestionIndex = -1;
+        }
+    }
+
+    // reapply filter on suggestions after each new input
+    $effect(() => {
+        if (searchSuggestions !== undefined) {
+            let possibleSuggestions = searchSuggestions(inputText.trim().toLowerCase());
+            suggestions = possibleSuggestions.filter((suggestion) => !items.includes(suggestion));
+        }
+    });
+</script>
+
+<!--
+@component
+Input component with chips showing all given input.
+
+If the user provides an input and set it (using ",", "+", "Tab" or "Enter"), a chip will be added
+in the input field showing this input. The chip can be deleted either by clicking on the cross or
+pressing "Backspace". The user can navigate in the chips using the left and right arrow keys.
+
+Customizations:
+- `label` to provide a label, which will be positioned at `labelPosition` ("top" or "left") relative to the input
+- `placeholder` to show a placeholder text in the input field
+- `searchSuggestions` to propagate a function, which search for a input string and return a string array,
+representing an input text, which should be searched in a list of possible suggestions.
+
+If the method `searchSuggestions` is given, the user gets a list of possible inputs beneath the
+input field, which can be selected by either clicking on the suggestions or add it like a normal item.
+Furthermore, the user can navigate in the suggestions using the up and down arrow keys.
+
+Usage:
+```svelte
+    <ChipsInput
+        bind:items={membersInput}
+        label="Members"
+        searchSuggestions={filterPossibleMembers}
+    />
+```
+-->
+<div class="flex flex-col w-full gap-2">
+    <div class="flex {labelPosition === 'top' ? 'flex-col gap-2' : 'flex-row gap-4 items-center'}">
+        <Label for={"chips-input-" + label}>{label}</Label>
+        <div
+            id={"chips-input-" + label}
+            class="flex flex-wrap w-full items-center gap-2.5 px-4 {items.length === 0
+                ? 'py-2'
+                : 'py-1.5'}
+    text-default border-input-border-slate bg-background rounded-md border overflow-x-auto"
```

Comment:
The indentation here is a bit weird.
```suggestion
                text-default border-input-border-slate bg-background rounded-md border overflow-x-auto"
```

(I still prefer using `cn` for that, but that's up to you 😄)